validate.go

```diff
@@ -32,7 +32,7 @@ func fieldValidators(model reflect.Value, opt ...Option) (map[string]validator,
 		model.Kind() == reflect.Pointer && model.Elem().Kind() != reflect.Struct:
 		return nil, fmt.Errorf("%s: model must be a struct or a pointer to a struct: %w", op, ErrInvalidParameter)
 	}
-	var m reflect.Value = model
+	m := model
```

Comment:
Fixes:
```
validate.go:35:8: ST1023: should omit type reflect.Value from declaration; it will be inferred from the right-hand side (staticcheck)
	var m reflect.Value = model
	      ^
```